libvast/include/vast/arrow_extension_types.hpp

```diff
@@ -69,14 +69,19 @@ class enum_extension_type : public arrow::ExtensionType {
 
 /// Address representation as an Arrow extension type.
 /// Internal (physical) representation is a 16-byte fixed binary.
-class address_extension_type : public arrow::ExtensionType {
+class ip_extension_type : public arrow::ExtensionType {
 public:
+  // NOTE: The identifier for the extension type of VAST's ip type has not
+  // changed when the type was renamed from address to ip because that would be
+  // a breaking change. This is fixable by registering two separate extension
+  // types with the same functionality but different ids, but that's a lot of
+  // effort for something users don't usually see.
```

Comment:
We're making breaking changes with 3.0, so isn't the time now to do this change?

Reply:
I wouldn't consider updating the internal representation of the extension type worth the requireed effort. If you think it is then please create do this in a follow-up PR. This will only ever be visible to devs.

Reply:
I wrote a follow-up story for this.